config-templates/ibm/ibm-vpc-block-csi-driver/changelog.json

```diff
@@ -3,23 +3,26 @@
         "5.0": {
             "status": "supported",
             "latest-revision": {
-                "revision": "5",
-                "release-date": "06 February 2023",
-                "cves": "CVE-2022-40303, CVE-2022-40304",
+                "revision": "6",
+                "release-date": "20 February 2023",
+                "cves": "CVE-2022-47629",
                 "golang": "1.18.9",
-                "ubi": "8.7-1049",
+                "ubi": "8.7-1049.1675784874",
                 "new-features": [],
-                "fixes": []
+                "fixes": [
+                   "Priorityclass added in deployment for controller and node server pods",
```

Comment:
Not necessary, but we can change `Priorityclass added in deployment for controller and node server pods` ---> `Priorityclass added for controller and node server pods`